routing/route/route.go

```diff
@@ -340,15 +340,19 @@ func (r *Route) ToSphinxPath() (*sphinx.PaymentPath, error) {
 func (r *Route) String() string {
 	var b strings.Builder
 
+	amt := r.TotalAmount
 	for i, hop := range r.Hops {
 		if i > 0 {
-			b.WriteString(",")
+			b.WriteString(" -> ")
```

Comment:
can you add an example of the old vs new output? Looks like fees aren't explicitly printed now?

Reply:
new output:
`1727264597517467648 (5001005 mSAT) -> 1727817651866959873 (5000000 mSAT), cltv 1608341`

The fees aren't explictly printed anymore, but can still be calculated by subtracting the last amount from the first amount. I am seeking for minimal log spam but still providing everything we need for debugging

Reply:
sgtm